### PR TITLE
feat(curriculum): add blog workshop 

### PR DESCRIPTION
**Pull request by jdwilkin4**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Comment by Ksound22** [comment]
I will get to this before the end of the week.

Well done 👍🏾

File: curriculum/challenges/english/25-front-end-development/workshop-blog-page/669fcb5106eeb13ab7afcd09.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+title: Step 5
+challengeType: 0
+dashedName: step-5
+---
+
+# --description--
+
+The first section you will build out is the page <dfn>header</dfn>. 
+
+The `header` element is used represent introductory content like page navigation and other introductory information.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-blog-page/669fcb5106eeb13ab7afcd09.md:12]:
```suggestion
The `header` element is used to represent introductory content like page navigation and other introductory information. 
```

File: curriculum/challenges/english/25-front-end-development/workshop-blog-page/669fcb06c3034a39f5431a38.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+id: 669fcb06c3034a39f5431a38
+title: Step 4
+challengeType: 0
+dashedName: step-4
+---
+
+# --description--
+
+To prepare to create some actual content, add a `body` element below the `head` element.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-blog-page/669fcb06c3034a39f5431a38.md:10]:
```suggestion
To prepare creating some actual content, add a `body` element below the `head` element.
```

**Review by Ksound22 — COMMENTED** [review]
I left two comments. OTher than that, things look good!

File: curriculum/challenges/english/25-front-end-development/workshop-blog-page/669fdfc9a5e3da42d2376609.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+title: Step 13
+challengeType: 0
+dashedName: step-13
+---
+
+# --description--
+
+Now that you have added the about section, try clicking on the About link to see the page jump down to that section.
+
+The next section in the blog page will be a list of posts talking about Mr.Whiskers.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/workshop-blog-page/669fdfc9a5e3da42d2376609.md:12]:
```suggestion
The next section in the blog page will be a list of posts talking about Mr. Whiskers.
```

**Review by zairahira — COMMENTED** [review]
The project is looking great Jessica.

I was wondering if the correct usage is `double check` or `double-check`?

File: curriculum/challenges/english/25-front-end-development/workshop-blog-page/669fdfc9a5e3da42d2376609.md (hunk 3 above)
Comment (jdwilkin4) [reply curriculum/challenges/english/25-front-end-development/workshop-blog-page/669fdfc9a5e3da42d2376609.md:12]: @zairahira 

According to cambridge and miraim dictionary it is double-check
https://dictionary.cambridge.org/us/dictionary/english/double-check

Comment (zairahira) [reply curriculum/challenges/english/25-front-end-development/workshop-blog-page/669fdfc9a5e3da42d2376609.md]: Not compulsory though, but we can change `double check` to `double-check` through out the workshop.

**Review by zairahira — APPROVED** [review]

**Review by Ksound22 — APPROVED** [review]

**Pull request merged**